### PR TITLE
Enhancement/account page mui refactor

### PR DESCRIPTION
**Pull request by Ak1yamaKiyoshi**

### CODE REVIEW

File: src/components/pages/account-page/components/security-tab/components/change-password-form/ChangePasswordForm.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Box } from '@mui/material';
+import { Box, Typography, useMediaQuery } from '@mui/material';
```
Comment (Yevheniy2020) [inline src/components/pages/account-page/components/security-tab/components/change-password-form/ChangePasswordForm.tsx:2]: you need replace legacy components to mui

File: src/components/pages/account-page/components/security-tab/components/change-password-form/ChangePasswordForm.tsx [hunk 2]
```diff
@@ -67,51 +69,53 @@ const ChangePasswordForm = () => {
...
+          <Form>
+            <Input
+              label="Поточний пароль"
+              placeholder="введи свій пароль"
+              type={InputType.PASSWORD}
+              name="oldPassword"
+              isSuccessOnDefault={true}
+            />
+            {errors.oldPassword === 'Введений пароль недійсний' && (
+              <Typography variant="body1">
```
Comment (Yevheniy2020) [inline src/components/pages/account-page/components/security-tab/components/change-password-form/ChangePasswordForm.tsx:82]: check mui documentation, tag p is not body1

File: src/components/pages/account-page/components/group-tab/GroupTab.tsx [hunk 3]
```diff
@@ -63,12 +62,16 @@ const GroupTab: FC = () => {
...
   if (!data || !user?.group || !user?.group.role) return null;
-  //TODO refactor this
   return (
-    <div>
-      <div>
-        <h4>Список групи {user.group.code}</h4>
-      </div>
+    <Box>
+      <Box>
+        <Typography variant="h4Bold">Список групи {user.group.code}</Typography>
```
Comment (Yevheniy2020) [inline src/components/pages/account-page/components/group-tab/GroupTab.tsx:68]: image

change to h4

File: src/components/pages/account-page/components/security-tab/components/change-password-form/ChangePasswordForm.tsx (hunk 1 above)
Comment (Ak1yamaKiyoshi) [reply src/components/pages/account-page/components/security-tab/components/change-password-form/ChangePasswordForm.tsx:2]: which one? 

Box & Typografy is already imported from mui,
Input works with formik, we don't have mui formik components
Custom link - uses mui

Comment (Yevheniy2020) [reply src/components/pages/account-page/components/security-tab/components/change-password-form/ChangePasswordForm.tsx:2]: Input , check components -> form -> formik -> input

**Review by Yevheniy2020 — APPROVED** [review]

**Pull request merged**